scalr/analysis/dge_lmem.py

```diff
@@ -195,9 +196,8 @@ def get_result_mxmodel_per_gene(self, gene: str,
                             f'{self.fixed_effect_column}[T.{category}]']
                 pickle.dumps(result_dict_per_gene)
                 return result_dict_per_gene
-        except Exception as e:
-            logger.warning(f"Error found in gene {gene}: {e}")
-            logger.warning(traceback.format_exc())
+        except LinAlgError as e:
+            return None
```

Comment:
By default the function return `None`. You can simply write `pass`.

Reply:
Do we not want to inform user that there is some issue & hence we are raising exception? Why returning None or passing the except case.

Reply:
Because of singular matrix, getting 'LinAlgError' for some random genes, and to remove these genes we are catching this exception so the calculation for other genes shouldn't terminate. In multiprocessing, adding logger to log these error resulting 'logger can't be pickled error', can't add print statement either.